src/routes/lobby/components/LobbyPlayerIndicator.vue

```diff
@@ -154,13 +158,32 @@ export default {
   z-index: 1;
 }
 
+.reverseWater {
+  width: 3600px;
+  height: 800px;
+  background-image: url('/img/waves.svg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  animation: waving-reverse 3s;
+  animation-fill-mode: forwards;
+  z-index: 1;
```

Comment:
In order to DRY this up, how about pulling everything except the animation itself into a style block shared by `.water` and `.reverseWater` since the rest is the same. We could either do that by introducing a new class shared in both cases, or by adding a style block with `.water, .reverseWater`

Reply:
That was actually my mistake, the `reverseWater` class is only there to trigger the animation! Because is the same component that has the `water` class properties. I will upload the fix right away